usr/sbin/truenas-install

```diff
@@ -405,7 +424,9 @@ partition_disks()
 
     _disksparts=$(for _disk in ${_disks}; do
 	create_partitions ${_disk} >&2
-	echo /dev/${_disk}3
+    # Passing the disk and partition to partitionLabeler for proper name format.
+    partitionLabeler _disk 3
+	echo /dev/${_partition}
```

Comment:
Please have a peek at 428 and then 429, you'll notice a mismatch of spaces/tabs. Can you please fix that ?